crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -176,10 +163,10 @@ pub struct AlgorithmUpdaterV1 {
     pub da_p_component: i64,
     /// The D component of the PID control for the DA gas price
     pub da_d_component: i64,
-    /// The average profit over the last `avg_window` blocks
-    pub profit_avg: i64,
-    /// The number of blocks to consider when calculating the average profit
-    pub avg_window: u32,
+    /// The last profit
+    pub last_profit: i64,
+    /// The profit before last
+    pub last_last_profit: i64,
```

Comment:
Also I think we can improve the naming here:
```suggestion
    pub last_profit: i64,
    /// The second last profit
    pub second_last_profit: i64,
```